modules/nf-core/sequali/tests/main.nf.test

```diff
@@ -0,0 +1,163 @@
+nextflow_process {
+
+    name "Test Process SEQUALI"
+    script "../main.nf"
+    process "SEQUALI"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "sequali"
+
+    test("sarscov2 - fastq single-end [sequali]") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:true], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert process.out.html[0][1] ==~ ".*/test.html"},
+                { assert process.out.json[0][1] ==~ ".*/test.json"},
+                { assert snapshot(process.out.versions).match() }
```

Comment:
You do not snapshot the files in themselves.
Is it due to some time stamp being created ?

Reply:
We tried the snapshot option, and match fails due to the md5 difference. Do you have any idea why this happens or how to handle it differently?

Reply:
What often happen is that there is a time and date written somewhere.
It might be complicated to spot in the html but the json should be easy to look at.
Could you run it and look inside the json ?
The json could be filtered and then snapshot 😉

Reply:
Thank you @LouisLeNezet so much for the help!

Reply:
We run the sequali with the same dataset twice. We compared JSON files and they contains **report_generated** field where the time is specified.

Reply:
You can filter I think the json with:
```nextflow
path(process.out.json[0][1]).json.key != "meta"
```

Reply:
As it is suggested in [nf-test: Example assertions](https://nf-co.re/docs/contributing/nf-test/assertions#snapshot-selective-portion-of-a-file) we extracted a part of the JSON file that is stable. We extracted values under the **summary** key and compared them.